docs/_guides/gke.md

```diff
@@ -0,0 +1,174 @@
+---
+layout: default
+---
+
+# Setting up Dispatch on GKE
+
+Google Container Engine (GKE) provides fully managed hosted kubernetes clusters.  This is great environment for running
+Dispatch, especially if you'd like Dispatch exposed to the internet.
+
+This guide walks through an installation of Dispatch on GKE with certificates and authorization enabled.  It's a good
+idea to enable authorization as otherwise the Dispatch API is unprotected and on the internet.  Addtionally, enabling
+certificates via letsencrypt enables use-cases which require signed certificates (like slack integrations).
+
+The certificate support is pretty experiemental at this point and requires AWS Route53.  Support for other DNS services
+is coming.
+
+## Create GKE cluster
+
+```
+export CLUSTER_NAME=dispatch-demo
+gcloud container clusters create -m n1-standard-2 --cluster-version 1.9.6-gke.1 ${CLUSTER_NAME}
+```
+
+## Fetch cluster credentials
+
+```
+gcloud container clusters get-credentials ${CLUSTER_NAME}
+```
+
+Kubectl should now be configured to the new cluster:
+
+```
+$ kubectl config current-context
+gke_dispatch-193801_us-west1-c_dispatch-demo
+```
+
+## Setup Helm and Tiller
+
+```
+kubectl create clusterrolebinding tiller-cluster-admin --clusterrole=cluster-admin --serviceaccount=kube-system:default
+```
+
+The following command it to work around a known [helm](https://github.com/kubernetes/helm/issues/3409)
+[issues](https://github.com/kubernetes/helm/issues/3379).  If it fails, retry:
+
+```
+helm init --tiller-image powerhome/tiller:git-3b22ecd --wait
```

Comment:
This shouldn't be needed with helm 2.8.2, as both of those issues were addressed (and it shouldn't require retry either). I will update our CI to use 2.8.2.

Reply:
good to know.